shared/listing.js

```diff
@@ -26,7 +27,7 @@ export const LISTING_SORT_BY = Object.freeze({
 
 export const DEFAULT_DAY_INTERVAL = 31
 export const MINIMUM_DAY_INTERVAL = 1
-export const MAXIMUM_BID_DAY_INTERVAL = 6 * 31 // six month
+export const MAXIMUM_BID_DAY_INTERVAL = 182 // six month
```

Comment:
```suggestion
export const MAXIMUM_BID_DAY_INTERVAL = 182 // six months
```

😛 

Also why this change?